### PR TITLE
Add models & break cyclic dependency between model and ONSEnvironment

### PR DESCRIPTION
**Pull request by jonmorphic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +26,11 @@ Your Swagger definition lives here:
 http://localhost:8080/party-api/1.0.0/swagger.json
 ```
 
+## Database
+
+The database will automatically be created when starting the application. It is also possible to drop/create the
+database via create_db.py. Run `python create_db.py` in the project root to see further guidance.
```
Comment (LJBabbage) [inline README.md]: python create_db.py needs changing now

File: swagger_server/database.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+def drop(config, engine, logger):
+    db_connection = config.get('db_connection')
+    db_schema = config.get('db_schema')
+
+    from swagger_server.models_local.base import Base
+    # fix-up the postgres schema:
+    Base.metadata.schema = db_schema if db_connection.startswith('postgres') else None
+
+    from swagger_server.models_local import _model
```
Comment (LJBabbage) [inline swagger_server/database.py:15]: As discussed one of these needs to change, ref line 33

**Review by LJBabbage — CHANGES_REQUESTED** [review]
LGTM, bar a couple of minor changes

File: swagger_server/database.py (hunk 2 above)
Comment (jonmorphic) [reply swagger_server/database.py:15]: Fixed. This is likely to change again in a subsequent PR as ras-party-v2 drifts away from code generator.

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**